Tests/test_Affy.py

```diff
@@ -69,17 +69,15 @@ def testAffy3(self):
             self.assertAlmostEqual(record.AlgorithmParameters["OutlierHigh"], 1.500)
             self.assertAlmostEqual(record.AlgorithmParameters["OutlierLow"], 1.004)
             self.assertEqual(record.AlgorithmParameters["AlgVersion"], "6.0")
-            self.assertEqual(record.AlgorithmParameters["FixedCellSize"], True)
+            self.assertTrue(record.AlgorithmParameters["FixedCellSize"])
             self.assertEqual(record.AlgorithmParameters["FullFeatureWidth"], 7)
             self.assertEqual(record.AlgorithmParameters["FullFeatureHeight"], 7)
-            self.assertEqual(
-                record.AlgorithmParameters["IgnoreOutliersInShiftRows"], False
-            )
-            self.assertEqual(record.AlgorithmParameters["FeatureExtraction"], True)
+            self.assertFalse(record.AlgorithmParameters["IgnoreOutliersInShiftRows"])
+            self.assertTrue(record.AlgorithmParameters["FeatureExtraction"])
             self.assertEqual(record.AlgorithmParameters["PoolWidthExtenstion"], 2)
             self.assertEqual(record.AlgorithmParameters["PoolHeightExtension"], 2)
-            self.assertEqual(record.AlgorithmParameters["UseSubgrids"], False)
-            self.assertEqual(record.AlgorithmParameters["RandomizePixels"], False)
+            self.assertFalse(record.AlgorithmParameters["UseSubgrids"])
+            self.assertFalse(record.AlgorithmParameters["RandomizePixels"])
```

Comment:
These five examples might be best left as is (with the special comment ``# noqa: A502`` to tell flake8-assertive to ignore them).

My thinking is we want to check the parser used the literal value ``True``, not that it used some value which evaluates as true (like a non-empty string or list).

Anyone want to give a second opinion here please?